test/halmos/HalmosTest.sol

```diff
@@ -74,17 +95,9 @@ contract HalmosTest is SymTest, Test {
             address borrower = svm.createAddress("borrower");
             args = abi.encode(marketParams, borrower, assets, shares, emptyData);
         } else if (selector == morpho.flashLoan.selector) {
-            uint256 rand = svm.createUint256("rand");
-            address token;
-            if (rand == 0) {
-                token = address(loanToken);
-            } else if (rand == 1) {
-                token = address(collateralToken);
-            } else {
-                ERC20Mock otherToken = new ERC20Mock();
-                token = address(otherToken);
-            }
```

Comment:
explicit case splitting is no longer needed. when the call target address is symbolic, all available addresses are automatically considered.